docs/source-pytorch/advanced/post-training quantization.rst

```diff
@@ -0,0 +1,158 @@
+
+**Audience** : Machine learning engineers optimizing models for a better model inference speed and lower memory usage
+
+
+Model Quantization
+==================
+Model quantization is an efficient model optimization tool that can accelerate the model inference speed and decrease the memory load while still maintaining the model accuracy.
+
+Different from the inherent model quantization callback ''QuantizationAwareTraining'' in Pytorch Lightning, Intel Neural Compressor provides a convenient model quantization API to quantize the already-trained Lightning module with Post-training Quantization and Quantization Aware Training. This extension API exhibits the merits of an ease-of-use coding environment and multi-functional quantization options. The user can easily quantize their fine-tuned model by adding a few clauses to their original code.  We only introduce post-training quantization in this document.
+
+There are two post-training quantization types in Intel Neural Compressor, post-training static quantization and post-training dynamic quantization.  Post-training dynamic quantization is a recommended starting point because it provides reduced memory usage and faster computation without additional calibration datasets. This type of quantization statically quantizes only the weights from floating point to integer at conversion time. This optimization provides latencies close to post-training static quantization. But the outputs of ops are still stored with the floating point, so the increased speed of dynamic-quantized ops is less than a static-quantized computation.
+
+Post-training static quantization saves the output of ops via INT8 bit. It can tackle the accuracy and latency loss caused by ''quant'' and ''dequant'' operations. For Post-training static quantization, the user needs to estimate the min-max range of all FP32 tensors in the model. Unlike constant tensors such as weights and biases, variable tensors such as model input, activations and model output cannot be calibrated unless the model run a few inference cycles. As a result, the converter requires a calibration dataset to estimate that range. This dataset can be a small subset (default 100 samples) of the training or the validation data.
+
+
+Installation
+============
+
+Prerequisites
+-------------
+Python version: 3.7, 3.8, 3.9, 3.10
+
+Install Dependencies
+--------------------
+::
+
+    pip install numpy pandas pyyaml scikit-learn scikit-image matplotlib schema py-cpuinfo contextlib2 requests Flask Flask-Cors Flask-SocketIO gevent gevent-websocket psutil Pillow pycocotools-windows; sys_platform != 'linux' pycocotools; sys_platform == 'linux' opencv-python prettytable cryptography sqlalchemy==1.4.27 alembic==1.7.7 pywin32; sys_platform != 'linux' deprecated

```

Comment:
Aren't these getting installed through

`pip install neural-compressor`
? 
We wouldn't want the user to install these manually.